worker/src/files.js

```diff
@@ -0,0 +1,50 @@
+const axios = require('axios');
+const FormData = require('form-data');
+const https = require('https');
+const { downloadFile } = require('./s3');
+const { File } = require('./models');
+
+const fileStatuses = {
+  uploading: 'UPLOADING',
+  uploaded: 'UPLOADED',
+  uploadFailed: 'UPLOAD_FAILED',
+  queued: 'SCANNING_QUEUED',
+  queueingFailed: 'QUEUEING_FAILED',
+  scanning: 'SCANNING',
+  approved: 'APPROVED',
+  rejected: 'REJECTED',
+};
+
+const getIdFromKey = async (key) => {
+  const file = await File.findOne({ where: { key } });
+  return file.dataValues.id;
+};
+
+const updateFileStatus = async (key, fileStatus) => {
+  const fileId = await getIdFromKey(key);
+  await File.update({ status: fileStatus }, { where: { id: fileId } });
+};
+
+const processFile = (key) => downloadFile(key).then((data) => {
```

Comment:
key is a uuid generated by the app. It is always equal to s3FileName. It also serves as a unique key in the queue. Since it defines a job, there won't be a job if key is null. I will add some code comments clarifying.